source/replication/server-side-replication-resynchronize-remote.rst

```diff
@@ -0,0 +1,321 @@
+
+   .. tab-item:: Replication Remote User
+
+      The following policy provides permissions for enabling synchronization of
+      replicated data *into* the cluster. Use the :mc-cmd:`mc admin policy add`
+      to add this policy to *both* deployments.
+
+      .. literalinclude:: /extra/examples/ReplicationRemoteUserPolicy.json
+         :class: copyable
+         :language: json
+
+      - The ``"EnableReplicationOnBucket"`` statement grants permission for 
+        a remote target to retrieve bucket-level configuration for supporting
+        replication operations on *all* buckets in the MinIO cluster. To
+        restrict the policy to specific buckets, specify those buckets as an
+        element in the ``Resource`` array similar to
+        ``"arn:aws:s3:::bucketName"``.
+
+      - The ``"EnableReplicatingDataIntoBucket"`` statement grants permission
+        for a remote target to synchronize data into *any* bucket in the MinIO
+        cluster. To restrict the policy to specific buckets, specify those 
+        buckets as an element in the ``Resource`` array similar to 
+        ``"arn:aws:s3:::bucketName/*"``.
+
+      Use the :mc-cmd:`mc admin policy add` to add this policy to *both*
+      deployments. You can then create a user on both deployments using
+      :mc-cmd:`mc admin user add` and associate the policy to those users
+      with :mc-cmd:`mc admin policy set`.
+
+MinIO strongly recommends creating users specifically for supporting 
+bucket replication operations. See 
+:mc:`mc admin user` and :mc:`mc admin policy` for more complete
+documentation on adding users and policies to a MinIO cluster.
+
+Existing Object Replication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Resynchronization requires :ref:`existing object replication
+<minio-replication-behavior-existing-objects>`. Specifically, resynchronization
+only applies to those replication configuration rules created where
+:mc-cmd-option:`~mc replicate add replicate` includes ``"existing-objects"``.
+
+Use :mc-cmd:`mc replicate ls` to list the replication rules for a bucket and
+verify which rules have existing object replication enabled.
+
+Considerations
+--------------
+
+Resynchronization Requires Time
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Resynchronization is a background processes that continually checks objects in
+the source MinIO bucket and copies them to the remote as-needed. The time
+required for replication to complete may vary depending on the number and size
+of objects, the throughput to the remote MinIO deployment, and the load on the
+source MinIO deployment. Total time for completion is generally not predictable
+due to these variables.
+
+MinIO recommends configuring load balancers or proxies to direct traffic only
+to the healthy cluster until synchronization completes. The following commands
+can provide insight into the resynchronization status:
+
+- :mc-cmd:`mc replicate status` on the source and remote to track total 
+  replicated data.
```

Comment:
`mc replicate status` on source will not reflect the state of resync because it computes numbers based on change in replication status. Forcible resync of already replicated objects will not show up in the status, and is not tracked separately as of now. 

To track resync progress, the `mc replicate status` on remote should show the total bytes received so far.

Reply:
Got it - left another comment noting some of the fuzziness around this approach, but it's something for now.